src/components/ProgressStep/__test__/progressStep.spec.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```

Comment:
All test that use the ProgressIndicator goes in the ProgressIndicator tests so move it.
Here only put the test that use the ProgressStep isolated